cortx-monitorQuickstartGuide.md

```diff
@@ -137,23 +137,83 @@ This guide provides a step-by-step walkthrough for getting you CORTX-Monitor Ser
     -rw-r--r-- 1 root root   9592 Aug 18 07:27 cortx-libsspl_sec-method_pki-1.0.0-1_git8907300.el7.x86_64.rpm
     -rw-r--r-- 1 root root 101004 Aug 18 07:27 cortx-libsspl_sec-devel-1.0.0-1_git8907300.el7.x86_64.rpm
     ```
-
-3. Install CORTX-Monitor RPMs
+    
+5. Install CORTX-Monitor RPMs
 
    You'll need to create a repository and copy all the required RPMs to it. Run the commands:
 
     ```shell
-    
     $ mkdir MYRPMS
```

Comment:
Let this be a full directory path.
`mkdir -p /root/MYRPMS`

Reply:
Done